cluster-provision/gocli/opts/prometheus/manifests/README.md

```diff
@@ -0,0 +1,33 @@
+# Prometheus
+
```

Comment:
They are copied yes
They are introduced in the opts package commit
since each opt depends on its manifests and conf to run. so it makes sense to add the code and the dependencies together